tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,75 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Box, Section } from "../components";
+import { Window } from "../layouts";
+
+interface CrewCreditsData {
+  groups: GroupBlockProps[]
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600} theme={"paper"}>
+      <Window.Content scrollable>
+        {data.groups?.map((group, index) =>
+          (data.groups[index].crew.length > 0 ? <GroupBlock key={index} group={group.group} crew={group.crew} /> : "")
+        )}
+      </Window.Content>
+    </Window>);
+};
+
+interface GroupBlockProps {
+  group: string,
+  crew: CrewMemberProps[]
+}
+
+const GroupBlock = (props:GroupBlockProps) => {
+  const { group, crew } = props;
+  const group_title = group;
+  return (
+    <Section title={group_title}>
+      {crew?.map((member, index) =>
+        (
+          member.head ? <CrewMember
```

Comment:
better to use `!!condition && (<Stuff>)` rather than a ternary I think

Reply:
[396c98d](https://github.com/goonstation/goonstation/pull/12339/commits/396c98d15b7468481a97050a3d360500f5c6f80f)